getsentry/sentry
# ref(replays): Implement the useVirtualizedList hook on the Replays Dom tab

The `useVirtualizedList` is a little helper to make it easier to update the react-virtualized component and CellMeasurer cache whenever the filtered list of items changes.

This change adds it to the DOM tab to clean things up a little, and help keep this related code together.

**Test Plan:**
I found a replay with lots of dom events, enough so that scrolling and searching through them was interesting.

Basically I did some searches and tried to make sure that any short/tall rows would have their sizes re-measured whenever the list changed.

1. Do a search for `<a role=\"button\"` to filter the list
2. Remove the search term from the textbox.

1. Do a search for a specific classname, so the search results don't scroll
2. Reload the page
3. Remove the search term from the textbox.

Everything seemed fine.
